frontend/testing/cypress/src/integration/studio/resourceadm.js

```diff
@@ -41,4 +41,132 @@ context('Resourceadm', () => {
+  });
+
+  it('is possible to visit Resource page and add a policy', () => {
+    cy.findByRole('table')
+      .findByRole('button', {
+        name: texts['resourceadm.dashboard_table_row_edit'],
+      })
+      .click();
+    cy.findByRole('button', {
+      name: texts['resourceadm.left_nav_bar_policy'],
+    }).click();
+    cy.findByRole('heading', {
+      name: texts['resourceadm.resource_navigation_modal_title_resource'],
+    });
+    cy.findByRole('button', {
+      name: texts['resourceadm.resource_navigation_modal_button_move_on'],
+    }).click();
+    cy.url().should('include', '/policy');
+    cy.findByRole('heading', {
+      name: texts['resourceadm.policy_editor_title'],
+    });
+    cy.findByRole('button', {
+      name: texts['policy_editor.card_button_text'],
+    }).click();
+    cy.findByRole('button', {
+      name: texts['resourceadm.left_nav_bar_back'],
+    }).click();
+    cy.findByRole('table').contains(texts['resourceadm.dashboard_table_row_has_policy']);
+    cy.url().should('include', '/ttd/ttd-resources');
+  });
+
+  it('is possible to visit Policy page and return to Resource page', () => {
+    cy.findByRole('table')
+      .findByRole('button', {
+        name: texts['resourceadm.dashboard_table_row_edit'],
+      })
+      .click();
+    cy.findByRole('button', {
+      name: texts['resourceadm.left_nav_bar_policy'],
+    }).click();
+    cy.findByRole('heading', {
+      name: texts['resourceadm.resource_navigation_modal_title_resource'],
+    });
+    cy.findByRole('button', {
+      name: texts['resourceadm.resource_navigation_modal_button_move_on'],
+    }).click();
+    cy.url().should('include', '/policy');
+    cy.findByRole('button', {
+      name: texts['resourceadm.left_nav_bar_about'],
+    }).click();
+    cy.findByRole('heading', {
+      name: texts['resourceadm.resource_navigation_modal_title_policy'],
+    });
+    cy.findByRole('button', {
+      name: texts['resourceadm.resource_navigation_modal_button_move_on'],
+    }).click();
+    cy.url().should('include', '/about');
+  });
+
+  it('is possible to visit Deploy page and return to Resource page', () => {
+    cy.findByRole('table')
+      .findByRole('button', {
+        name: texts['resourceadm.dashboard_table_row_edit'],
+      })
+      .click();
+    cy.findByRole('button', {
+      name: texts['resourceadm.left_nav_bar_deploy'],
+    }).click();
+    cy.findByRole('heading', {
+      name: texts['resourceadm.resource_navigation_modal_title_resource'],
+    });
+    cy.findByRole('button', {
+      name: texts['resourceadm.resource_navigation_modal_button_move_on'],
+    }).click();
+    cy.url().should('include', '/deploy');
+    cy.findByRole('heading', {
+      name: texts['resourceadm.deploy_title'],
+    });
+
+    // Note: cy.findByRole('button') will not work while Error-module is visible
+    cy.get('button').contains('Om ressursen').click();
```

Comment:
Kan "Om ressursen" erstattes med en referanse til en tekstnøkkel?

Reply:
Ja, den bør erstattes med getByRole("button" & tekstnøkkel...) sånn som de andre.

Men det fungerer ikke her, fordi det finnes for tiden, en annen button, i Error-module, med samme tekstnøkkel, som da forstyrrer getByRole(). 

Jeg tenkte på å lage en data-cy tag for LeftNavBar knappene, for å skille dem, men det er jo ikke helt opp til standard heller. 

Så jeg gikk for gammeldags get().contains() ettersom det fungerte. Så får vi heve standarden i neste runde.

Reply:
Er dette kanskje noe som kan løses med [within](https://docs.cypress.io/api/commands/within)? Uansett, det jeg siktet til var bare å erstatte selve teksten, så det blir slik: `cy.get('button').contains(texts[...]).click();`

Reply:
Ah! Da misforsto jeg. Ja, å sette inn tekstnøkkel i get.().contains() virker. 

Så skal jeg se på within() til neste runde. Takk for tips.